src/silo_api/api.cpp

```diff
@@ -20,6 +20,23 @@
 #include "silo_api/logging.h"
 #include "silo_api/request_handler_factory.h"
 
+std::string preprocessingConfigPath(const Poco::Util::AbstractConfiguration& config) {
+   if (config.hasProperty("preprocessingConfig")) {
+      return config.getString("preprocessingConfig");
+   }
+   SPDLOG_DEBUG("preprocessingConfig not found in config file. Using default value: config.yaml");
+   return "preprocessingConfig.yaml";
+}
+
+std::string databaseConfigPath(const Poco::Util::AbstractConfiguration& config) {
+   if (config.hasProperty("databaseConfig")) {
+      return config.getString("databaseConfig");
+   }
+   SPDLOG_DEBUG("databaseConfig not found in config file. Using default value: databaseConfig.yaml"
```

Comment:
I have seen that we generally used snake_case instead of camelCase for these files. Maybe we should change the default? (I know that I introduced it in the wrong way in a previous PR)